src/components/ProcessCreate/Census/Spreadsheet/index.tsx

```diff
@@ -130,18 +129,13 @@ export const CensusCsvManager = () => {
                     }
                   }}
                   onBlur={onBlur}
-                  ref={ref}
+                  name={'weightedVote'}
```

Comment:
The way DetailedCheckbox is done does not require to have a Controller wrapping it AFAIK.